### PR TITLE
added cancel button and event emitter for it

### PR DESCRIPTION
**Pull request by Favouroked**

### CODE REVIEW

**Review by oneEyedSunday — COMMENTED** [review]
I see you emit `cancel`, but how does this actually cancel the form?

**Comment by Favouroked** [comment]
> I see you emit `cancel`, but how does this actually cancel the form?

the consumer handles the cancel, once it emits, the route is changed to the client page

**Comment by oneEyedSunday** [comment]
ok

**Review by radiumrasheed — COMMENTED** [review]
@oneEyedSunday @Favouroked I believe the use of the cancel button should be to undo changes made to the form.
so it might be ideal to store a state of the data after the view inits
then reset it back to this value when the cancel button is clicked
then you can still choose to emit after that

_I'll merge PR for now_

File: src/wizard.component.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ import {WizardStepComponent} from './wizard-step.component';
 				<button type="button" class="btn btn-secondary float-left" (click)="previous()" [hidden]="!hasPrevStep || !activeStep.showPrev">
 					Previous
 				</button>
+                <button type="button" class="btn btn-danger float-right" (click)="cancelForm()">Cancel</button>
```
Comment (radiumrasheed) [inline src/wizard.component.ts]: `float-left` would be ideal

**Review by radiumrasheed — APPROVED** [review]
LGTM

**Pull request merged**